### PR TITLE
Updating Agones version to 1.8

### PR DESCRIPTION
**Pull request by aLekSer**
Terraform 0.12.6 and above should be used. Tested with Terraform 0.13.
Fixed destroy.sh script to work with Helm 3.

### CODE REVIEW

**Comment by markmandel** [comment]
Since 1.8 comes out next Tuesday, should we just jump ahead to 1.8?

Also, now we're on helm 3, can we remove the whole destroy.sh entirely?

**Comment by markmandel** [comment]
We can now push this to 1.8 :smile:

**Comment by aLekSer** [comment]
That's true, will update this PR after testing.

**Comment by aLekSer** [comment]
Just met this issue:
https://github.com/hashicorp/terraform-provider-helm/issues/509
It seems that they are connected with files left after switching from Helm2 to Helm3 terraform configs.

**Comment by aLekSer** [comment]
This version was tested and ready for Review. Note that `agones-tf` was used as directory name due to aforementioned bug in Helm Terraform provider.

File: terraform-4-cluster/main.tf [hunk 1]
```diff
@@ -62,7 +62,7 @@ module "game-cluster-us-1" {
 }
 
 module "game-cluster-us-2" {
-  source  = "./agones"
+  source  = "./agones-tf"
```
Comment (markmandel) [inline terraform-4-cluster/main.tf:65]: Some other ideas:
- `agones-cluster`
- `cluster`
-`gameserver-cluster` 

WDYT?

Comment (aLekSer) [reply terraform-4-cluster/main.tf:65]: Yes, proposed versions are more verbose and meaningful.

Comment (aLekSer) [reply terraform-4-cluster/main.tf]: Updated

**Comment by aLekSer** [comment]
Performing a test `terraform apply` right now after rebasing on latest master.

**Pull request merged**